src/main/java/org/fundacion/jala/converter/core/VideoConverter.java

```diff
@@ -153,4 +159,12 @@ public void setOutputFileName(final String newOutputFileName) {
     public String getOutputFileName() {
         return outputFileName;
     }
+
+    /**
+     * Returns the object result for the operation.
```

Comment:
there should be a space between the lines 164 and 165

Reply:
Solved.